src/autoscaler.ts

```diff
@@ -34,6 +34,12 @@ export default class AutoscaleProcessor {
         const currentInventory = await this.jibriTracker.getCurrent(group.name);
         const count = currentInventory.length;
 
+        const scalingAllowed = await this.jibriTracker.allowScaling(group.name);
```

Comment:
This is OK for now, but we may decide to put this in a separate use of the redis client by the autoscaler class, rather than using jibriTracker for all redis-related activities.